src/js/configuration/index.js

```diff
@@ -2,22 +2,56 @@ const readConfig = key => window._env_[key] || process.env["REACT_APP_" + key];
 
 const conf = {
   API: {
-    BASE_URL: readConfig("BASE_API_URL")
+    base_url: readConfig("BASE_API_URL"),
+  },
+  APP: {
+    max_instances: readConfig("INSTANCES_MAX"),
+    dns: readConfig("DNS"),
+    git_path: readConfig("GIT_PATH"),
+    password: readConfig("PASSWORD"),
+    grafana_uri: readConfig("GRAFANA_BASE_URI")
+  },
+  ENDPOINTS: {
+    salon_rc: readConfig("SALON_ROCKETCHAT")
+  },
+  KUBERNETES: {
+    kub_kc_client_id: readConfig("KUB_KC_CLIENT_ID"),
+    kub_kc_url: readConfig("KUB_KC_URL"),
+    kub_api_server: readConfig("KUB_APISERVER_URL")
+  },
+  VAULT: {
+    vault_base_uri: readConfig("VAULT_BASE_URI")
   },
   CHAT: {
-    URL: readConfig("CHAT_URL")
+    url: readConfig("CHAT_URL")
   },
   AUTHENTICATION: {
     TYPE: readConfig("AUTH_TYPE"),
     OIDC: {
       realm: readConfig("AUTH_OIDC_REALM"),
-      "auth-server-url": readConfig("AUTH_OIDC_URL"),
-      "ssl-required": readConfig("AUTH_OIDC_SSL_REQUIRED"),
+      auth_server_url: readConfig("AUTH_OIDC_URL"),
```

Comment:
I think the keycloak parameters must be `-` separated and not `_` separated. We have no control over that as they are passed directly to the Keycloak constructor